### PR TITLE
Delay creation of AttributeCollectionSpec.

### PR DESCRIPTION
**Pull request by toregge**
@geirst : please review

### CODE REVIEW

File: searchcore/src/vespa/searchcore/proton/server/fast_access_doc_subdb.h [hunk 1]
```diff
@@ -92,14 +92,15 @@ class FastAccessDocSubDB : public StoreOnlyDocSubDB
...
 
-    std::unique_ptr<AttributeCollectionSpec> createAttributeSpec(const AttributesConfig &attrCfg, const AllocStrategy& alloc_strategy, std::optional<SerialNum> serialNum) const;
     AttributeManager::SP getAndResetInitAttributeManager();
     virtual IFlushTargetList getFlushTargetsInternal() override;
     void reconfigureAttributeMetrics(const IAttributeManager &newMgr, const IAttributeManager &oldMgr);
 
     IReprocessingTask::UP createReprocessingTask(IReprocessingInitializer &initializer,
                                                  const std::shared_ptr<const document::DocumentTypeRepo> &docTypeRepo) const;
 
+    bool get_fast_access_attributes_only() const noexcept { return _fastAccessAttributesOnly; }
```
Comment (geirst) [inline searchcore/src/vespa/searchcore/proton/server/fast_access_doc_subdb.h:102]: Consider rename to _has_fast_access_attributes_only()_.

**Review by geirst — APPROVED** [review]
👍

File: searchcore/src/vespa/searchcore/proton/server/fast_access_doc_subdb.h (hunk 1 above)
Comment (toregge) [reply searchcore/src/vespa/searchcore/proton/server/fast_access_doc_subdb.h]: Done.

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**